### PR TITLE
build: release 2.6.0

### PR DESCRIPTION
**Pull request by andre-code**
New version of the UI

/deploy renku=000-fix-acceptance-tests #persist

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+## [2.6.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.5.0...2.6.0) (2022-07-26)
+
+### Features
+
+* add confirmation when changing a value in the project settings ([#1922](https://github.com/SwissDataScienceCenter/renku-ui/issues/1922), [#1945](https://github.com/SwissDataScienceCenter/renku-ui/issues/1945))
+* add visibility tooltip on the project card ([#1916](https://github.com/SwissDataScienceCenter/renku-ui/issues/1916), [#1940](https://github.com/SwissDataScienceCenter/renku-ui/issues/1940))
+* add support for mermaid format in markdown view ([#1807](https://github.com/SwissDataScienceCenter/renku-ui/issues/1807), [#1941](https://github.com/SwissDataScienceCenter/renku-ui/issues/1941))
+* change font for Inter, update button and card styles ([#1885](https://github.com/SwissDataScienceCenter/renku-ui/issues/1885), [#1953](https://github.com/SwissDataScienceCenter/renku-ui/issues/1953))
+* include robots.txt ([#1948](https://github.com/SwissDataScienceCenter/renku-ui/issues/1948))
+* include sitemap.xml ([#1917](https://github.com/SwissDataScienceCenter/renku-ui/issues/1917), [#1948](https://github.com/SwissDataScienceCenter/renku-ui/issues/1948))
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md:12]:
```suggestion
```
I would remove these since they are not user-facing changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+* include renku social media links in landing page ([#1923](https://github.com/SwissDataScienceCenter/renku-ui/issues/1923), [#1938](https://github.com/SwissDataScienceCenter/renku-ui/issues/1938))
+* Include renku dataset import, renku rollback, and remove renku autosave and saving progress section in cheatsheet ([#1961](https://github.com/SwissDataScienceCenter/renku-ui/issues/1961))
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md:14]:
```suggestion
* update cheatsheet ([#1961](https://github.com/SwissDataScienceCenter/renku-ui/issues/1961))
```
Let's simplify this

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
 # Changes
 
+## [2.6.0](https://github.com/SwissDataScienceCenter/renku-ui/compare/2.5.0...2.6.0) (2022-07-26)
+
+### Features
+
+* add confirmation when changing a value in the project settings ([#1922](https://github.com/SwissDataScienceCenter/renku-ui/issues/1922), [#1945](https://github.com/SwissDataScienceCenter/renku-ui/issues/1945))
+* add visibility tooltip on the project card ([#1916](https://github.com/SwissDataScienceCenter/renku-ui/issues/1916), [#1940](https://github.com/SwissDataScienceCenter/renku-ui/issues/1940))
+* add support for mermaid format in markdown view ([#1807](https://github.com/SwissDataScienceCenter/renku-ui/issues/1807), [#1941](https://github.com/SwissDataScienceCenter/renku-ui/issues/1941))
+* change font for Inter, update button and card styles ([#1885](https://github.com/SwissDataScienceCenter/renku-ui/issues/1885), [#1953](https://github.com/SwissDataScienceCenter/renku-ui/issues/1953))
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.md:10]:
```suggestion
* update layout: buttons, cards, font, colors ([#1885](https://github.com/SwissDataScienceCenter/renku-ui/issues/1885), [#1953](https://github.com/SwissDataScienceCenter/renku-ui/issues/1953))
```

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
Thanks for preparing this!
I suggested some rewording. Otherwise, ready to go.

File: CHANGELOG.md (hunk 1 above)
Comment (andre-code) [reply CHANGELOG.md]: sure

File: CHANGELOG.md (hunk 2 above)
Comment (andre-code) [reply CHANGELOG.md]: @lorenzo-cavazzi  Thank you for your review, all suggestions were implemented.

**Review by lorenzo-cavazzi — APPROVED** [review]
Great, let's merge it! :rocket:

**Pull request merged**